docs/docs/howto/installation/configure-klaw-wizard.rst

```diff
@@ -0,0 +1,27 @@
+Configure Klaw with Wizard
+==========================
+
+When Klaw is installed for the first time, it prompts the Administrators with a Wizard to configure and make it available for the users.
+
+.. note::  Wizard is accessible at : `https://<host>:<port>/helpwizard`
+
+.. image:: /../../../_static/images/KlawWizard.png
+
+1. Configure Kafka clusters and update Klaw Cluster Api properties with relevant config
+
+2. Configure Environments
+
+3. Configure Tenant and Environment hierarchy
+
+4. Test Kafka connectivity
+
+5. For existing Kafka clusters, migrate Topics and Acls to Klaw.
+
+6. Create Teams
+
+7. Create Users
+
+8. Create any other Roles and Permissions (Optional)
```

Comment:
The way this is structured right now it feels like it's either too few or too much. Let me explain, in here only the steps of the wizard are described, but using the "number points list" feels like each point should contain more information, maybe what can be tuned, or what are the preferred configs.

If the previous one is not the intended, I think we could change the format of this list to something less vertically spacious (so it doesn't feel like you are going to describe each of them).

Reply:
> The way this is structured right now it feels like it's either too few or too much. Let me explain, in here only the steps of the wizard are described, but using the "number points list" feels like each point should contain more information, maybe what can be tuned, or what are the preferred configs.
> 
> If the previous one is not the intended, I think we could change the format of this list to something less vertically spacious (so it doesn't feel like you are going to describe each of them).

I could probably help with refining the content to make it sound better for the end user.

Reply:
thanks @harshini-rangaswamy

Reply:
Agree, each of them are elaborated now.